activemodel/CHANGELOG.md

```diff
@@ -1,2 +1,20 @@
+*   Backport `ActiveRecord::Normalization` to `ActiveModel::Attributes::Normalization`
+
+    ```ruby
+    class User
+      include ActiveModel::Model
```

Comment:
`include ActiveModel::Model` isn't actually necessary, right?  I think we can omit it.

Actually, the same goes for the examples in the method docs.